src/Orleans.Core/Runtime/OutsideRuntimeClient.cs

```diff
@@ -145,9 +147,7 @@ internal void ConsumeServices(IServiceProvider services)
 
             try
             {
-                //init logger for UnobservedExceptionsHandlerClass
-                UnobservedExceptionsHandlerClass.InitLogger(this.loggerFactory);
-                if (!UnobservedExceptionsHandlerClass.TrySetUnobservedExceptionHandler(UnhandledException))
+                if (!unobservedExceptionsHandler.TrySetUnobservedExceptionHandler(UnhandledException))
```

Comment:
I meant to remove this for a long time. The client should not subscribe to unobserved exceptions, as it is not the owner of the host, it's just a client sdk running in a frontend. Several people already complained about Orleans logging all unrelated unobserved exceptions with our logger.

Reply:
+1 The original rational here IIRC was that this would help us investigate customer issue by logging unhandled exceptions. I agree with @jdom that we should remove it. It doesn't add much of real value at the cost of confusing people.

Reply:
sounds good